engine/src/main/java/com/redhat/cloud/notifications/templates/DrawerTemplateMigrationService.java

```diff
@@ -1,351 +0,0 @@
-package com.redhat.cloud.notifications.templates;
-
-import com.cronutils.utils.StringUtils;
-import com.redhat.cloud.notifications.models.EventType;
-import com.redhat.cloud.notifications.models.IntegrationTemplate;
-import com.redhat.cloud.notifications.models.Template;
-import io.quarkus.logging.Log;
-import jakarta.enterprise.context.ApplicationScoped;
-import jakarta.inject.Inject;
-import jakarta.persistence.EntityManager;
-import jakarta.persistence.NoResultException;
-import jakarta.transaction.Transactional;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.UncheckedIOException;
-import java.time.LocalDateTime;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
-
-import static com.redhat.cloud.notifications.events.IntegrationDisabledNotifier.INTEGRATION_DISABLED_EVENT_TYPE;
-import static com.redhat.cloud.notifications.processors.email.aggregators.AdvisorEmailAggregator.*;
-import static java.nio.charset.StandardCharsets.UTF_8;
-
-@ApplicationScoped
-public class DrawerTemplateMigrationService {
```

Comment:
Will drawer templates no longer be added into the DB after this PR?

Reply:
That was a comment for myself. I didn't mean to submit it because I'm sure I'll find the answer while finishing the review.